students/chaycasso/Lesson02/gridprinter.py

```diff
@@ -0,0 +1,79 @@
+#Grid Printer Exercise
+#Chay Casso, 1/29/18
+
+def grid1():
+    """Draw a simple 2x2 grid composed of 5x5 squares."""
+    plus = "+"
+    minus = "-"
+    blank = " "
+    pipe = "|"
+    blankminus = blank + minus
+    print(plus + blankminus * 4 + blank + plus + blankminus * 4 + blank + plus)
+    for i in range(1, 5):
+        print(pipe + blank * 9 + pipe + blank * 9 + pipe)
+    print(plus + blankminus * 4 + blank + plus + blankminus * 4 + blank + plus)
+    for i in range(1, 5):
+        print(pipe + blank * 9 + pipe + blank * 9 + pipe)
+    print(plus + blankminus * 4 + blank + plus + blankminus * 4 + blank + plus)
+
+
+def GridLine2(gridsize=3):
```

Comment:
functions names should never start with a capital letter, and the words should be divided by a `_` .Everything should be lower case unless its a constant variable
https://www.python.org/dev/peps/pep-0008/